src/vs/workbench/contrib/notebook/test/notebookViewModel.test.ts

```diff
@@ -18,22 +18,22 @@ import { reduceCellRanges, ICellRange } from 'vs/workbench/contrib/notebook/brow
 
 suite('NotebookViewModel', () => {
 	const instantiationService = new TestInstantiationService();
-	const blukEditService = instantiationService.get(IBulkEditService);
+	const bulkEditService = instantiationService.get(IBulkEditService);
```

Comment:
Don't change vs stuff unless you have to. Just ignore these spelling errors

Reply:
You can always make the change in the vscode repo itself, but yes agreed, here is not the place to change that.